app/src/main/java/org/schabi/newpipe/info_list/holder/StreamGridInfoItemHolder.java

```diff
@@ -2,11 +2,17 @@
 
 import android.view.ViewGroup;
 
+import androidx.annotation.Nullable;
```

Comment:
You also need to remove this unsued import:
```
Error: eckstyle] [ERROR] /home/runner/work/NewPipe/NewPipe/app/src/main/java/org/schabi/newpipe/info_list/holder/StreamGridInfoItemHolder.java:5:8: Unused import - androidx.annotation.Nullable. [UnusedImports]
```

Reply:
thanks)